Woof/Woof/Helper/Color+Extension.swift

```diff
@@ -0,0 +1,10 @@
+import SwiftUI
+
+/// Describes the color scheme using in the application
+extension Color {
+    static let whiteCustom = Color(red: 251 / 255, green: 251 / 255, blue: 251 / 255)
```

Comment:
Do you need tests right now to avoid unintentional changes?

Reply:
As discussed, yes, we should test cases when someone can change the values.